### PR TITLE
Update `mailto:` handler details in readme

### PR DESCRIPTION
**Pull request by mit-mit**
Update based on new info in https://github.com/flutter/flutter/issues/10032

### CODE REVIEW

File: packages/url_launcher/README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Common schemes supported by both iOS and Android:
 | Scheme | Example | Action |
 |---|---|---|
 | `http:<URL>` , `https:<URL>` | `http://flutter.io` | Open URL in the default browser |
-| `mailto:<email address>` | `mailto:smith@example.org` | Open <email address> in the default email app |
+| `mailto:<email address>?subject=<subject>&body<body> | `mailto:smith@example.org?subject=News&body=New%20%Flutter%plugin` | Create email to <email address> in the default email app |
```
Comment (szakarias) [inline packages/url_launcher/README.md]: Change 'mailto:smith@example.org?subject=News&body=New%20%Flutter%plugin' to 'mailto:smith@example.org?subject=News&body=New%20Flutter%20plugin'

**Comment by mit-mit** [comment]
@szakarias please review latest commit adding some best practice notes on `canLaunch`

File: packages/url_launcher/README.md [hunk 2]
```diff
@@ -46,9 +47,23 @@ Common schemes supported by both iOS and Android:
...
 More details can be found here for [iOS](https://developer.apple.com/library/content/featuredarticles/iPhoneURLScheme_Reference/Introduction/Introduction.html) and [Android](https://developer.android.com/guide/components/intents-common.html)
 
+## Handling missing URL receivers
+
+A particular mobile device may not be able to receive all supported URL schemes.
+For example, a tablet may not have cellular support and thus not support
+launching a URL using the `sms` scheme, and a device may not have an email app
+and thus not launching a URL using the `email` scheme.
+
+We recommend checking which URL are supported using the
```
Comment (szakarias) [inline packages/url_launcher/README.md]: URL schemes

File: packages/url_launcher/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+## [0.4.2+2] - 2017-08-15
```
Comment (szakarias) [inline packages/url_launcher/CHANGELOG.md]: Maybe update the date?

**Pull request merged**